packages/plus/src/components/SteppedListCard/SteppedListContent.tsx

```diff
@@ -22,15 +22,27 @@ type SteppedListContentProps = {
   children: ((nextStep: (completed: boolean) => void) => ReactNode) | ReactNode
   image?: ReactNode
   stepNumber: number
+  completed?: boolean
 }
 export const SteppedListContent = ({
   subHeader,
   children,
   image,
   stepNumber,
+  completed = false,
 }: SteppedListContentProps) => {
   const containerData = useContext(Data)
 
+  useEffect(() => {
+    containerData.setDone(prevDone => {
+      const updatedDone = [...prevDone]
+      updatedDone[stepNumber - 1] = completed
+
+      return updatedDone
+    })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
using this flag is source of side effects, it doesn't work if you put all the dependencies?